pkg/reconciler/apis/apibinding/apibinding_reconcile_test.go

```diff
@@ -139,11 +139,11 @@ var (
 
 	someOtherWidgetsAPIResourceSchema = &apisv1alpha1.APIResourceSchema{
 		ObjectMeta: metav1.ObjectMeta{
-			Name: "another.widgets.other.io",
```

Comment:
What's the meaning of this change?

Reply:
This is meant to be the conflicting schema, and it needs to be in the same API group. The conflict test cases passed before because they were relying on the getCRD behavior defined below to return a widgets.kcp.dev CRD (the conflict checker now uses a schema instead of a CRD).